src/icloudpd/base.py

```diff
@@ -970,7 +970,7 @@ def download_photo_(counter: Counter, photo: PhotoAsset) -> bool:
                                 download.set_utime(download_path, created_date)
                             logger.info("Downloaded %s", truncated_path)
 
-                if xmp_sidecar:
+                if xmp_sidecar and download_size != AssetVersionSize.ADJUSTED:
```

Comment:
Yeah, it might make sense to think this through better. Maybe only download the xmp for original? Generally since the XMP is based on the asset record, it should only be created once for a specific foto.